example_site/src/components/TemplateBrowser.js

```diff
@@ -32,23 +32,26 @@ function Template({ template }) {
   );
 }
 
-function DurationFilter({
+function Filter({
   filter,
+  filterKey,
   templateFilter,
   setTemplateFilter,
 }) {
-  const [buttonStatus, setButtonStatus] = useState(true);
-  const buttonColor = templateFilter.duration === filter.value ? theBlue : 'black';
+  const [filterIsApplied, setFilterIsApplied] = useState(true);
```

Comment:
This is super nitpicky and you had no way of knowing, but we typically start boolean values w/ is, should, or has, so let's change this to `[isFilterApplied, setIsFilterApplied`.